test/cli.py

```diff
@@ -3,7 +3,8 @@
 """
 
 import os
```

Comment:
Missing tests for 
```
benchit my_model1.py my_model2.py OR
benchit my_model[1-2].py  OR
benchit my_model*.py OR
```

Or is the idea that those are "secret features" and not really supported?

Reply:
Or at least `benchit models/*/*.py` right? That's a feature we plan to use.

Reply:
As we talked about this morning, as long as we rely that bash regular expressions, there are no additional tests needed.